{{cookiecutter.project_name}}/LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Michael Herman
+Copyright (c) 2020 {{cookiecutter.project_name}}
```

Comment:
Hey! This looks awesome but could you please generate the year automatically? Yo can use something like this:

`Copyright (c) {% now 'utc', '%Y' %} {{cookiecutter.project_name}}`

Reply:
Done!